Building Python package from PyPI sdist dies with CMake configuration errors

I'm trying to use `manifold3d` with Python on ARM64 linux and as there's no binary package it is building from source. Unfortunately this dies with:

```console
$ pip3 install --verbose manifold3d
Using pip 24.0 from /media/psf/Home/Projects/visuals/flitter/.venv-linux/lib/python3.12/site-packages/pip (python 3.12)
Collecting manifold3d
  Using cached manifold3d-2.5.1.tar.gz (418 kB)
  Running command pip subprocess to install build dependencies
  Collecting scikit-build-core
    Using cached scikit_build_core-0.9.9-py3-none-any.whl.metadata (19 kB)
  Collecting packaging>=21.3 (from scikit-build-core)
    Using cached packaging-24.1-py3-none-any.whl.metadata (3.2 kB)
  Collecting pathspec>=0.10.1 (from scikit-build-core)
    Using cached pathspec-0.12.1-py3-none-any.whl.metadata (21 kB)
  Using cached scikit_build_core-0.9.9-py3-none-any.whl (152 kB)
  Using cached packaging-24.1-py3-none-any.whl (53 kB)
  Using cached pathspec-0.12.1-py3-none-any.whl (31 kB)
  Installing collected packages: pathspec, packaging, scikit-build-core
  Successfully installed packaging-24.1 pathspec-0.12.1 scikit-build-core-0.9.9
  Installing build dependencies ... done
  Running command Getting requirements to build wheel
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  Getting requirements to build wheel ... done
  Running command pip subprocess to install backend dependencies
  Collecting cmake>=3.18
    Using cached cmake-3.30.1-py3-none-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (6.1 kB)
  Collecting ninja>=1.5
    Using cached ninja-1.11.1.1-py2.py3-none-manylinux2014_aarch64.manylinux_2_17_aarch64.whl.metadata (5.3 kB)
  Using cached cmake-3.30.1-py3-none-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (26.3 MB)
  Using cached ninja-1.11.1.1-py2.py3-none-manylinux2014_aarch64.manylinux_2_17_aarch64.whl (155 kB)
  Installing collected packages: ninja, cmake
  Successfully installed cmake-3.30.1 ninja-1.11.1.1
  Installing backend dependencies ... done
  Running command Preparing metadata (pyproject.toml)
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  *** scikit-build-core 0.9.9 using CMake 3.30.1 (metadata_wheel)
  Preparing metadata (pyproject.toml) ... done
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/3a/be/650f9c091ef71cb01d735775d554e068752d3ff63d7943b26316dc401749/numpy-1.21.2.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/5f/d6/ad58ded26556eaeaa8c971e08b6466f17c4ac4d786cd3d800e26ce59cc01/numpy-1.21.3.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/fb/48/b0708ebd7718a8933f0d3937513ef8ef2f4f04529f1f66ca86d873043921/numpy-1.21.4.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/c2/a8/a924a09492bdfee8c2ec3094d0a13f2799800b4fdc9c890738aeeb12c72e/numpy-1.21.5.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/45/b7/de7b8e67f2232c26af57c205aaad29fe17754f793404f59c8a730c7a191a/numpy-1.21.6.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
Collecting numpy>=1.26.0b1 (from manifold3d)
  Obtaining dependency information for numpy>=1.26.0b1 from https://files.pythonhosted.org/packages/5e/e3/944b70438d3b7e2742fece7da8dfba6f7ef7dccdd163d1a613f7027f4d5b/numpy-2.0.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata
  Using cached numpy-2.0.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (62 kB)
Using cached numpy-2.0.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (13.6 MB)
Building wheels for collected packages: manifold3d
  Running command Building wheel for manifold3d (pyproject.toml)
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  *** scikit-build-core 0.9.9 using CMake 3.30.1 (wheel)
  *** Configuring CMake...
  loading initial cache file /tmp/tmpjvb_a5b3/build/CMakeInit.txt
  -- The CXX compiler identification is GNU 13.2.0
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /usr/bin/aarch64-linux-gnu-g++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  CMake Warning at manifoldDeps.cmake:4 (find_package):
    By not providing "FindClipper2.cmake" in CMAKE_MODULE_PATH this project has
    asked CMake to find a package configuration file provided by "Clipper2",
    but CMake did not find one.

    Could not find a package configuration file provided by "Clipper2" with any
    of the following names:

      Clipper2Config.cmake
      clipper2-config.cmake

    Add the installation prefix of "Clipper2" to CMAKE_PREFIX_PATH or set
    "Clipper2_DIR" to a directory containing one of the above files.  If
    "Clipper2" provides a separate development package or SDK, be sure it has
    been installed.
  Call Stack (most recent call first):
    CMakeLists.txt:68 (include)


  -- Checking for module 'Clipper2'
  --   Package 'Clipper2', required by 'virtual:world', not found
  -- Checking for module 'tbb'
  --   Package 'tbb', required by 'virtual:world', not found
  -- clipper2 not found, downloading from source
  -- The C compiler identification is GNU 13.2.0
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /usr/bin/aarch64-linux-gnu-gcc - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- tbb not found, downloading from source
  CMake Deprecation Warning at /tmp/tmpjvb_a5b3/build/_deps/tbb-src/CMakeLists.txt:15 (cmake_minimum_required):
    Compatibility with CMake < 3.5 will be removed from a future version of
    CMake.

    Update the VERSION argument <min> value or use a ...<max> suffix to tell
    CMake that the project does not need compatibility with older versions.


  CMake Warning at /tmp/tmpjvb_a5b3/build/_deps/tbb-src/CMakeLists.txt:119 (message):
    You are building oneTBB as a static library.  This is highly discouraged
    and such configuration is not supported.  Consider building a dynamic
    library to avoid unforeseen issues.


  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
  -- Found Threads: TRUE
  -- Checking for one of the modules 'hwloc'
  -- Looking for getcontext
  -- Looking for getcontext - found
  -- TBBBind build targets are disabled due to unsupported environment
  -- Found Python: /media/psf/Home/Projects/visuals/flitter/.venv-linux/bin/python3.12 (found version "3.12.3") found components: Interpreter Development.Module
  Parallel Backend: TBB
  CMake Error at bindings/CMakeLists.txt:16 (add_subdirectory):
    add_subdirectory given source "c" which is not an existing directory.


  Python_EXECUTABLE=/media/psf/Home/Projects/visuals/flitter/.venv-linux/bin/python3.12
  -- Configuring incomplete, errors occurred!

  *** CMake configuration failed
  error: subprocess-exited-with-error
  
  × Building wheel for manifold3d (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /media/psf/Home/Projects/visuals/flitter/.venv-linux/bin/python3.12 /media/psf/Home/Projects/visuals/flitter/.venv-linux/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /tmp/tmpuyuztmsz
  cwd: /tmp/pip-install-aa6abeuk/manifold3d_7439409543de4e61a6a0180913455ca8
  Building wheel for manifold3d (pyproject.toml) ... error
  ERROR: Failed building wheel for manifold3d
Failed to build manifold3d
ERROR: Could not build wheels for manifold3d, which is required to install pyproject.toml-based projects
```

I thought this might be a Linux issue, but then I tried forcing a build from source on Apple Silicon macOS and it fails in exactly the same way:

```console
$ pip3 install --verbose --no-binary manifold3d manifold3d
Using pip 24.0 from /Users/jonathan/Projects/visuals/flitter/.venv-manifold3d/lib/python3.12/site-packages/pip (python 3.12)
Collecting manifold3d
  Using cached manifold3d-2.5.1.tar.gz (418 kB)
  Running command pip subprocess to install build dependencies
  Collecting scikit-build-core
    Using cached scikit_build_core-0.9.9-py3-none-any.whl.metadata (19 kB)
  Collecting packaging>=21.3 (from scikit-build-core)
    Using cached packaging-24.1-py3-none-any.whl.metadata (3.2 kB)
  Collecting pathspec>=0.10.1 (from scikit-build-core)
    Using cached pathspec-0.12.1-py3-none-any.whl.metadata (21 kB)
  Using cached scikit_build_core-0.9.9-py3-none-any.whl (152 kB)
  Using cached packaging-24.1-py3-none-any.whl (53 kB)
  Using cached pathspec-0.12.1-py3-none-any.whl (31 kB)
  Installing collected packages: pathspec, packaging, scikit-build-core
  Successfully installed packaging-24.1 pathspec-0.12.1 scikit-build-core-0.9.9

  [notice] A new release of pip is available: 24.0 -> 24.2
  [notice] To update, run: pip install --upgrade pip
  Installing build dependencies ... done
  Running command Getting requirements to build wheel
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  Getting requirements to build wheel ... done
  Running command Preparing metadata (pyproject.toml)
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  *** scikit-build-core 0.9.9 using CMake 3.30.1 (metadata_wheel)
  Preparing metadata (pyproject.toml) ... done
  Link requires a different Python (3.12.4 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/3a/be/650f9c091ef71cb01d735775d554e068752d3ff63d7943b26316dc401749/numpy-1.21.2.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.4 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/5f/d6/ad58ded26556eaeaa8c971e08b6466f17c4ac4d786cd3d800e26ce59cc01/numpy-1.21.3.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.4 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/fb/48/b0708ebd7718a8933f0d3937513ef8ef2f4f04529f1f66ca86d873043921/numpy-1.21.4.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.4 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/c2/a8/a924a09492bdfee8c2ec3094d0a13f2799800b4fdc9c890738aeeb12c72e/numpy-1.21.5.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.4 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/45/b7/de7b8e67f2232c26af57c205aaad29fe17754f793404f59c8a730c7a191a/numpy-1.21.6.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
Collecting numpy>=1.26.0b1 (from manifold3d)
  Obtaining dependency information for numpy>=1.26.0b1 from https://files.pythonhosted.org/packages/c2/da/3d8debb409bc97045b559f408d2b8cefa6a077a73df14dbf4d8780d976b1/numpy-2.0.1-cp312-cp312-macosx_14_0_arm64.whl.metadata
  Using cached numpy-2.0.1-cp312-cp312-macosx_14_0_arm64.whl.metadata (60 kB)
Using cached numpy-2.0.1-cp312-cp312-macosx_14_0_arm64.whl (5.0 MB)
Building wheels for collected packages: manifold3d
  Running command Building wheel for manifold3d (pyproject.toml)
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  *** scikit-build-core 0.9.9 using CMake 3.30.1 (wheel)
  *** Configuring CMake...
  loading initial cache file /var/folders/wh/303d39cn41b2v9jbtqh2607h0000gn/T/tmprelbx19b/build/CMakeInit.txt
  -- The CXX compiler identification is AppleClang 15.0.0.15000309
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  CMake Warning at manifoldDeps.cmake:4 (find_package):
    By not providing "FindClipper2.cmake" in CMAKE_MODULE_PATH this project has
    asked CMake to find a package configuration file provided by "Clipper2",
    but CMake did not find one.

    Could not find a package configuration file provided by "Clipper2" with any
    of the following names:

      Clipper2Config.cmake
      clipper2-config.cmake

    Add the installation prefix of "Clipper2" to CMAKE_PREFIX_PATH or set
    "Clipper2_DIR" to a directory containing one of the above files.  If
    "Clipper2" provides a separate development package or SDK, be sure it has
    been installed.
  Call Stack (most recent call first):
    CMakeLists.txt:68 (include)


  -- Checking for module 'Clipper2'
  --   No package 'Clipper2' found
  -- Checking for module 'tbb'
  --   No package 'tbb' found
  -- clipper2 not found, downloading from source
  -- The C compiler identification is AppleClang 15.0.0.15000309
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- glm not found, downloading from source
  -- GLM: Version 1.0.1
  -- GLM: Disable -Wc++98-compat warnings
  -- GLM: Build with C++ features auto detection
  -- tbb not found, downloading from source
  CMake Deprecation Warning at /var/folders/wh/303d39cn41b2v9jbtqh2607h0000gn/T/tmprelbx19b/build/_deps/tbb-src/CMakeLists.txt:15 (cmake_minimum_required):
    Compatibility with CMake < 3.5 will be removed from a future version of
    CMake.

    Update the VERSION argument <min> value or use a ...<max> suffix to tell
    CMake that the project does not need compatibility with older versions.


  CMake Warning at /var/folders/wh/303d39cn41b2v9jbtqh2607h0000gn/T/tmprelbx19b/build/_deps/tbb-src/CMakeLists.txt:119 (message):
    You are building oneTBB as a static library.  This is highly discouraged
    and such configuration is not supported.  Consider building a dynamic
    library to avoid unforeseen issues.


  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
  -- Found Threads: TRUE
  -- Checking for one of the modules 'hwloc'
  -- Looking for getcontext
  -- Looking for getcontext - found
  -- TBBBind build targets are disabled due to unsupported environment
  -- Found Python: /Users/jonathan/Projects/visuals/flitter/.venv-manifold3d/bin/python3.12 (found version "3.12.4") found components: Interpreter Development.Module
  Parallel Backend: TBB
  thrust not found, downloading from source
  CMake Warning (dev) at /opt/homebrew/Cellar/cmake/3.30.1/share/cmake/Modules/FetchContent.cmake:1953 (message):
    Calling FetchContent_Populate(Thrust) is deprecated, call
    FetchContent_MakeAvailable(Thrust) instead.  Policy CMP0169 can be set to
    OLD to allow FetchContent_Populate(Thrust) to be called directly for now,
    but the ability to call it with declared details will be removed completely
    in a future version.
  Call Stack (most recent call first):
    src/utilities/CMakeLists.txt:30 (FetchContent_Populate)
  This warning is for project developers.  Use -Wno-dev to suppress it.

  CMake Error at bindings/CMakeLists.txt:16 (add_subdirectory):
    add_subdirectory given source "c" which is not an existing directory.


  Python_EXECUTABLE=/Users/jonathan/Projects/visuals/flitter/.venv-manifold3d/bin/python3.12
  -- Configuring incomplete, errors occurred!

  *** CMake configuration failed
  error: subprocess-exited-with-error
  
  × Building wheel for manifold3d (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /Users/jonathan/Projects/visuals/flitter/.venv-manifold3d/bin/python3.12 /Users/jonathan/Projects/visuals/flitter/.venv-manifold3d/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /var/folders/wh/303d39cn41b2v9jbtqh2607h0000gn/T/tmp21iikswr
  cwd: /private/var/folders/wh/303d39cn41b2v9jbtqh2607h0000gn/T/pip-install-t7hwy22w/manifold3d_92abf52db6af417e941012d6c95638d6
  Building wheel for manifold3d (pyproject.toml) ... error
  ERROR: Failed building wheel for manifold3d
Failed to build manifold3d
ERROR: Could not build wheels for manifold3d, which is required to install pyproject.toml-based projects
```

It seems to be something broken in the PyPI source distribution *only* as if I clone this repo and do an install from there it works fine:

```console
$ git clone --recurse-submodules https://github.com/elalish/manifold.git
Cloning into 'manifold'...
remote: Enumerating objects: 11206, done.
remote: Counting objects: 100% (938/938), done.
remote: Compressing objects: 100% (497/497), done.
remote: Total 11206 (delta 611), reused 646 (delta 401), pack-reused 10268
Receiving objects: 100% (11206/11206), 36.03 MiB | 2.23 MiB/s, done.
Resolving deltas: 100% (7393/7393), done.
Submodule 'bindings/python/third_party/nanobind' (https://github.com/wjakob/nanobind) registered for path 'bindings/python/third_party/nanobind'
Submodule 'src/third_party/quickhull' (https://github.com/akuukka/quickhull) registered for path 'src/third_party/quickhull'
Cloning into '/media/psf/Home/Projects/visuals/manifold/bindings/python/third_party/nanobind'...
remote: Enumerating objects: 7299, done.        
remote: Counting objects: 100% (106/106), done.        
remote: Compressing objects: 100% (72/72), done.        
remote: Total 7299 (delta 42), reused 73 (delta 28), pack-reused 7193        
Receiving objects: 100% (7299/7299), 2.58 MiB | 3.45 MiB/s, done.
Resolving deltas: 100% (5303/5303), done.
Cloning into '/media/psf/Home/Projects/visuals/manifold/src/third_party/quickhull'...
remote: Enumerating objects: 471, done.        
remote: Counting objects: 100% (48/48), done.        
remote: Compressing objects: 100% (33/33), done.        
remote: Total 471 (delta 20), reused 31 (delta 14), pack-reused 423        
Receiving objects: 100% (471/471), 147.08 KiB | 392.00 KiB/s, done.
Resolving deltas: 100% (283/283), done.
Submodule path 'bindings/python/third_party/nanobind': checked out '8d7f1ee0621c17fa370b704b2100ffa0243d5bfb'
Submodule 'ext/robin_map' (https://github.com/Tessil/robin-map) registered for path 'bindings/python/third_party/nanobind/ext/robin_map'
Cloning into '/media/psf/Home/Projects/visuals/manifold/bindings/python/third_party/nanobind/ext/robin_map'...
remote: Enumerating objects: 1142, done.        
remote: Counting objects: 100% (196/196), done.        
remote: Compressing objects: 100% (77/77), done.        
remote: Total 1142 (delta 130), reused 149 (delta 98), pack-reused 946        
Receiving objects: 100% (1142/1142), 883.78 KiB | 1.42 MiB/s, done.
Resolving deltas: 100% (777/777), done.
Submodule path 'bindings/python/third_party/nanobind/ext/robin_map': checked out '188c45569cc2a5dd768077c193830b51d33a5020'
Submodule path 'src/third_party/quickhull': checked out '1ffbc6f884ea1da89e104a5996cf8a726db673d5'
bother@ vox activate flitter/.venv-linux/
Activated "flitter/.venv-linux/".

$ cd manifold
$ pip3 install --verbose --editable .
Using pip 24.0 from /media/psf/Home/Projects/visuals/flitter/.venv-linux/lib/python3.12/site-packages/pip (python 3.12)
Obtaining file:///media/psf/Home/Projects/visuals/manifold
  Running command pip subprocess to install build dependencies
  Collecting scikit-build-core
    Using cached scikit_build_core-0.9.9-py3-none-any.whl.metadata (19 kB)
  Collecting packaging>=21.3 (from scikit-build-core)
    Using cached packaging-24.1-py3-none-any.whl.metadata (3.2 kB)
  Collecting pathspec>=0.10.1 (from scikit-build-core)
    Using cached pathspec-0.12.1-py3-none-any.whl.metadata (21 kB)
  Using cached scikit_build_core-0.9.9-py3-none-any.whl (152 kB)
  Using cached packaging-24.1-py3-none-any.whl (53 kB)
  Using cached pathspec-0.12.1-py3-none-any.whl (31 kB)
  Installing collected packages: pathspec, packaging, scikit-build-core
  Successfully installed packaging-24.1 pathspec-0.12.1 scikit-build-core-0.9.9
  Installing build dependencies ... done
  Running command Checking if build backend supports build_editable
  Checking if build backend supports build_editable ... done
  Running command Getting requirements to build editable
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  Getting requirements to build editable ... done
  Running command pip subprocess to install backend dependencies
  Collecting cmake>=3.18
    Using cached cmake-3.30.1-py3-none-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (6.1 kB)
  Collecting ninja>=1.5
    Using cached ninja-1.11.1.1-py2.py3-none-manylinux2014_aarch64.manylinux_2_17_aarch64.whl.metadata (5.3 kB)
  Using cached cmake-3.30.1-py3-none-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (26.3 MB)
  Using cached ninja-1.11.1.1-py2.py3-none-manylinux2014_aarch64.manylinux_2_17_aarch64.whl (155 kB)
  Installing collected packages: ninja, cmake
  Successfully installed cmake-3.30.1 ninja-1.11.1.1
  Installing backend dependencies ... done
  Running command Preparing editable metadata (pyproject.toml)
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  *** scikit-build-core 0.9.9 using CMake 3.30.1 (metadata_editable)
  Preparing editable metadata (pyproject.toml) ... done
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/3a/be/650f9c091ef71cb01d735775d554e068752d3ff63d7943b26316dc401749/numpy-1.21.2.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/5f/d6/ad58ded26556eaeaa8c971e08b6466f17c4ac4d786cd3d800e26ce59cc01/numpy-1.21.3.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/fb/48/b0708ebd7718a8933f0d3937513ef8ef2f4f04529f1f66ca86d873043921/numpy-1.21.4.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/c2/a8/a924a09492bdfee8c2ec3094d0a13f2799800b4fdc9c890738aeeb12c72e/numpy-1.21.5.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
  Link requires a different Python (3.12.3 not in: '>=3.7,<3.11'): https://files.pythonhosted.org/packages/45/b7/de7b8e67f2232c26af57c205aaad29fe17754f793404f59c8a730c7a191a/numpy-1.21.6.zip (from https://pypi.org/simple/numpy/) (requires-python:>=3.7,<3.11)
Collecting numpy>=1.26.0b1 (from manifold3d==2.5.1)
  Obtaining dependency information for numpy>=1.26.0b1 from https://files.pythonhosted.org/packages/5e/e3/944b70438d3b7e2742fece7da8dfba6f7ef7dccdd163d1a613f7027f4d5b/numpy-2.0.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata
  Using cached numpy-2.0.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (62 kB)
Using cached numpy-2.0.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (13.6 MB)
Building wheels for collected packages: manifold3d
  Running command Building editable for manifold3d (pyproject.toml)
  WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
  *** scikit-build-core 0.9.9 using CMake 3.30.1 (editable)
  *** Configuring CMake...
  loading initial cache file /tmp/tmpsc_9a58z/build/CMakeInit.txt
  -- The CXX compiler identification is GNU 13.2.0
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /usr/bin/aarch64-linux-gnu-g++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  -- Checking for module 'Clipper2'
  --   Package 'Clipper2', required by 'virtual:world', not found
  -- Checking for module 'tbb'
  --   Package 'tbb', required by 'virtual:world', not found
  -- clipper2 not found, downloading from source
  -- The C compiler identification is GNU 13.2.0
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /usr/bin/aarch64-linux-gnu-gcc - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- tbb not found, downloading from source
  CMake Deprecation Warning at /tmp/tmpsc_9a58z/build/_deps/tbb-src/CMakeLists.txt:15 (cmake_minimum_required):
    Compatibility with CMake < 3.5 will be removed from a future version of
    CMake.

    Update the VERSION argument <min> value or use a ...<max> suffix to tell
    CMake that the project does not need compatibility with older versions.


  CMake Warning at /tmp/tmpsc_9a58z/build/_deps/tbb-src/CMakeLists.txt:119 (message):
    You are building oneTBB as a static library.  This is highly discouraged
    and such configuration is not supported.  Consider building a dynamic
    library to avoid unforeseen issues.


  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
  -- Found Threads: TRUE
  -- Checking for one of the modules 'hwloc'
  -- Looking for getcontext
  -- Looking for getcontext - found
  -- TBBBind build targets are disabled due to unsupported environment
  -- Found Python: /media/psf/Home/Projects/visuals/flitter/.venv-linux/bin/python3.12 (found version "3.12.3") found components: Interpreter Development.Module
  Parallel Backend: TBB
  Python_EXECUTABLE=/media/psf/Home/Projects/visuals/flitter/.venv-linux/bin/python3.12
  -- Configuring done (29.8s)
  -- Generating done (0.0s)
  -- Build files have been written to: /tmp/tmpsc_9a58z/build
  *** Building project with Ninja...

[... about a hundred lines of stuff elided ...]  

  *** Installing project into wheel...
  -- Install configuration: "Release"
  -- Installing: /tmp/tmpsc_9a58z/wheel/platlib/manifold3d.cpython-312-aarch64-linux-gnu.so
  -- Set non-toolchain portion of runtime path of "/tmp/tmpsc_9a58z/wheel/platlib/manifold3d.cpython-312-aarch64-linux-gnu.so" to "/tmp/tmpsc_9a58z/wheel/platlib/lib"
  *** Making editable...
  *** Created manifold3d-2.5.1-cp312-cp312-linux_aarch64.whl...
  Building editable for manifold3d (pyproject.toml) ... done
  Created wheel for manifold3d: filename=manifold3d-2.5.1-cp312-cp312-linux_aarch64.whl size=1006684 sha256=c42edfb5b399268dbf318b62affcdc2e43283315b34d3c5bad2b311c5a7a6390
  Stored in directory: /tmp/pip-ephem-wheel-cache-v4ywjs8n/wheels/e4/27/5a/6ef941e5d4b6c8289abaa26f529af203b22da895393b6372b6
Successfully built manifold3d
Installing collected packages: numpy, manifold3d
  changing mode of /media/psf/Home/Projects/visuals/flitter/.venv-linux/bin/f2py to 775
  changing mode of /media/psf/Home/Projects/visuals/flitter/.venv-linux/bin/numpy-config to 775
Successfully installed manifold3d-2.5.1 numpy-2.0.1
```

This was the clue:

```
  CMake Error at bindings/CMakeLists.txt:16 (add_subdirectory):
    add_subdirectory given source "c" which is not an existing directory.
```

--- a/.github/workflows/build_wheels.yml
+++ b/.github/workflows/build_wheels.yml
@@ -1,6 +1,6 @@
 name: build_wheels
 
-on: 
+on:
   release:
     types: [published]
   workflow_dispatch:
@@ -35,6 +35,8 @@ jobs:
         submodules: recursive
     - name: Build SDist
       run: pipx run build --sdist
+    - name: Confirm SDist can be built
+      run: pip wheel dist/*.tar.gz
     - uses: actions/upload-artifact@v4
       with:
         name: python-sdist

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -44,7 +44,7 @@ sdist.exclude = [
     "bindings/python/third_party/nanobind/docs/",
 ]
 wheel.packages = ["manifold3d"]
-cmake.args = ["-DMANIFOLD_PYBIND=ON", "-DMANIFOLD_PAR=TBB", "-DMANIFOLD_TEST=OFF", "-DBUILD_SHARED_LIBS=OFF"]
+cmake.args = ["-DMANIFOLD_PYBIND=ON", "-DMANIFOLD_CBIND=OFF", "-DMANIFOLD_PAR=TBB", "-DMANIFOLD_TEST=OFF", "-DBUILD_SHARED_LIBS=OFF"]
 install.components = ["bindings"]
 
 [tool.cibuildwheel]

